lib/api/liskApi.js

```diff
@@ -428,18 +488,26 @@ LiskAPI.prototype.listMultisignatureTransactions = function (callback) {
  */
 
 LiskAPI.prototype.getMultisignatureTransaction = function (transactionId, callback) {
-	this.sendRequest('transactions/multisignatures/get', { id: transactionId }, function (result) {
+	this.sendRequest('transactions/unsigned', { id: transactionId }, function (result) {
 		return callback(result);
 	});
 };
 
+/**
+ * @method broadcastSignedTransaction
+ * @param transaction
+ * @param callback
+ *
+ * @return API object
+ */
+
 LiskAPI.prototype.broadcastSignedTransaction = function (transaction, callback) {
 
 	var request = {
 		requestMethod: 'POST',
 		requestUrl: privateApi.getFullUrl.call(this) + '/peer/' + 'transactions',
 		nethash: this.nethash,
-		requestParams: { transaction }
+		requestParams: { transaction: transaction }
 	};
 
 	privateApi.doPopsicleRequest.call(this, request).then(function (result) {
```

Comment:
Why don't we have a generic request method that abstracts some of this functionality for `POST` requests like we do for `GET` (or why isn't there one which covers both)?